CONTRIBUTING.md

```diff
@@ -67,7 +67,8 @@ It's possible to take only a subset of the screenshots which greatly speeds up t
 
 To do this follow the following steps:
 
-1. Change the `runOnly` property in [`Screenshots.swift`](./Example/Backpack Screenshot/Screenshots.swift)
+
+1. In [`Screenshots.swift`](./Example/Backpack\ Screenshot/Screenshots.swift) change the `runOnly` property per the guide.
```

Comment:
This link isn't rendering correctly

Reply:
Yeah, I'm not sure it can be fixed due to the space in the path :(